packages/gatsby-source-contentful/src/__tests__/extend-node-type.js

```diff
@@ -51,6 +51,7 @@ describe(`contentful extend node type`, () => {
         width: 450,
         height: 399,
         quality: 50,
+        background: `rgb:000000`,
```

Comment:
What's this for?

Reply:
The background is set when doing cropping operations where the source image does not completely cover the result.

See:
![](https://images.contentful.com/wln3fygpfpvt/7orLdboQQowIUs22KAW4U/6c709893ae83938bfef443a7e5f8c658/matt-palmer-254999.jpg?w=1180&h=480&fit=pad)
https://images.contentful.com/wln3fygpfpvt/7orLdboQQowIUs22KAW4U/6c709893ae83938bfef443a7e5f8c658/matt-palmer-254999.jpg?w=1180&h=480&fit=pad

![](https://images.contentful.com/wln3fygpfpvt/7orLdboQQowIUs22KAW4U/6c709893ae83938bfef443a7e5f8c658/matt-palmer-254999.jpg?w=1180&h=480&fit=pad&bg=rgb:000000)
https://images.contentful.com/wln3fygpfpvt/7orLdboQQowIUs22KAW4U/6c709893ae83938bfef443a7e5f8c658/matt-palmer-254999.jpg?w=1180&h=480&fit=pad&bg=rgb:000000

Reply:
Ah cool 👍